transcribed/page0890.txt

```diff
@@ -7,7 +7,7 @@
 007     051 004         SKIPGE IOBFT(Q)
 008     001 016         POPJ P,
 009     002 022         CONO PI,UTCOFF
-010     051 004         LDB J,[IOLO,,IOBFT(Q)
+010     051 004         LDB J,[IOLO,,IOBFT(Q)]
```

Comment:
was correct before. everything else is right though

Reply:
The MIDAS syntax is to put "literals" in between [ ] brackets, but the closing bracket is optional in early MIDAS.  PDP-6 code (such as this very example) heavily favors leaving out the optional close bracked.  This changed later when multi-line literals were implemented, which made the closing bracket non optional.